mlrun/feature_store/api.py

```diff
@@ -840,6 +841,8 @@ def _ingest_with_spark(
             df = source.filter_df_start_end_time(df, timestamp_key)
         if featureset.spec.graph and featureset.spec.graph.steps:
             df = run_spark_graph(df, featureset, namespace, spark)
+        if isinstance(df, Response) and df.status_code == 400:
+            mlrun.errors.raise_for_status_code(df.status_code, df.body.split(": ")[1])
```

Comment:
what is the split()[1] extracting? would it always work?

Reply:
Also because server.run behavior

Reply:
Also result of server.run fun behavior